setup/emc-update-impl.bash

```diff
@@ -127,6 +127,7 @@ then
     _git_clone_or_update https://github.com/husarion/rosbot_description.git "$EMC_SYSTEM_DIR"/src/rosbot_description
 else
     # Robot specific packages
+    _git_clone_or_update https://github.com/tue-robotics/rosbot_bringup.git "$EMC_SYSTEM_DIR"/src/rosbot_bringup
     _git_clone_or_update https://github.com/tue-robotics/mrc_hero_bringup "$EMC_SYSTEM_DIR"/src/mrc_hero_bringup
```

Comment:
Don't install both, when you only need one...

Reply:
I know, that's one of the (other) reasons it was still a draft. In order to select which to install we need to set the appropriate enviroment variables, which hasn't been done yet.